docs/generated/packages/nx/executors/run-commands.json

```diff
@@ -72,16 +72,19 @@
             },
             { "type": "string" }
           ]
-        }
+        },
+        "x-priority": "important"
       },
       "command": {
         "type": "string",
-        "description": "Command to run in child process."
+        "description": "Command to run in child process.",
+        "x-priority": "important"
```

Comment:
For properties that aren't "fully" required like this one, it's probably fine to keep the `important` property.